bin/all_sky_search/pycbc_add_statmap

```diff
@@ -98,10 +98,10 @@ logging.info('Combining foreground segments')
 
 # Convert segmentlistdict to a list ('seglists') of segmentlists
 # then np.sum(seglists, axis=0) does seglists[0] + seglists[1] + ...
-if len(indiv_segs) > 1:
-    foreground_segs = np.sum(list(indiv_segs.values()), axis=0)
-else:
-    foreground_segs = indiv_segs.values()[0]
```

Comment:
@GarethDaviesGW Can you check this. I think this is equivalent, but don't understand why this was complicated previously?

Reply:
I'm not sure why this was over-complicated in this way - I don't see how this wouldn't work

Reply:
and the proposed solution looks correct as well (This was probably early on in my understanding of segmentlists)

Reply:
The problem is that taking the length is actually wrong as it is a dict. It was meant to avoid the case where there is a single segment as there the bumpy sum will return a bumpy bday instead of a segment list..